### PR TITLE
adding the main editor file RELATES #12

### PR DESCRIPTION
**Pull request by IsraaMoqbel**

### CODE REVIEW

File: public/js/maineditor.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
+const select = function(selector) {
```
Comment (NoureldeanSaed) [inline public/js/maineditor.js]: The file should be named `mainEditor.js`

File: public/js/maineditor.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  })
+})
+
+
+addNewLinkBtn.addEventListener("click", function() {
+
+  const br = document.createElement("br");
+
+
+  if (flag === false) {
```
Comment (NoureldeanSaed) [inline public/js/maineditor.js]: `if (!flag)`
`else`

File: public/js/maineditor.js (hunk 1 above)
Comment (IsraaMoqbel) [reply public/js/maineditor.js]: Done!

File: public/js/maineditor.js (hunk 2 above)
Comment (IsraaMoqbel) [reply public/js/maineditor.js]: Done!

**Pull request merged**